VacaYAY/BusinessLogicLayer/Contracts/IPositionService.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BusinessLogicLayer.Contracts
+{
+    public interface IPositionService
```

Comment:
Why empty interface?

Reply:
No need for that. Let's be in the scope of the task

You broke [YAGNI ](https://martinfowler.com/bliki/Yagni.html) principle 😄